share/doc/homebrew/Maintainer-Guidelines.md

```diff
@@ -6,20 +6,118 @@ definitely not a beginner’s guide.
 
 Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
+----
+
+Quick links:
+  [General Guidelines](#general-guidelines) •
+  [Core and Core PRs](#core-and-core-prs) •
```

Comment:
That's the delimiter between the links. It is supposed to be a poor man's TOC and renders as:

> Quick links: General Guidelines • Core and Core PRs • Formulae and Formula PRs